invoke.go

```diff
@@ -193,7 +194,20 @@ func sendPost(ctx context.Context, route string, m InvokeMessage, t http.RoundTr
 			fmt.Printf("  %v: %v\n", k, v[0]) // NOTE len==1 value slices assumed
 		}
 	}
-	resp, err := client.PostForm(route, values)
+
+	req, err := http.NewRequestWithContext(ctx, "POST", route, bytes.NewBufferString(m.Data))
+	if err != nil {
+		return nil, "", fmt.Errorf("failure to create request: %w", err)
+	}
+
+	ct := "application/json"
```

Comment:
@lance seems the defaulting is handled somewhere else and the default value there is `text/plain` shall I change that?

Reply:
> @lance seems the defaulting is handled somewhere else and the default value there is `text/plain` shall I change that?

Yes, I think so.